### PR TITLE
Use more processors to build base builds.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by mijovic — DISMISSED** [review]
Looks good

File: .circleci/config.yml [hunk 1]
```diff
@@ -583,9 +586,11 @@ jobs:
       - store_artifacts: *artifacts_test_results
 
   b_ems:
+    resource_class: xlarge
     docker:
       - image: ethereum/solidity-buildpack-deps:emsdk-1.39.15-2
     environment:
+      NJOBS: 9
```
Comment (cameel) [inline .circleci/config.yml]: In https://github.com/ethereum/solidity/pull/9368#discussion_r454183010 @ekpyron suggested using `MAKEFLAGS` for that.

Comment (ekpyron) [reply .circleci/config.yml]: At least if it works, yeah :-).

File: scripts/travis-emscripten/build_emscripten.sh [hunk 2]
```diff
@@ -53,7 +53,7 @@ cmake \
   -DBoost_USE_STATIC_RUNTIME=1 \
   -DTESTS=0 \
   ..
-make -j 4 soljson
+make -j "${NJOBS:-4}" soljson
```
Comment (cameel) [inline scripts/travis-emscripten/build_emscripten.sh]: Maybe you should also modify Travis config to set this variable? Or is it just that you're fine with the default and don't want to change it to anything higher right now?

**Review by ekpyron — APPROVED** [review]
Yep, that seems nicer to me! We should confirm that it actually works as expected (i.e. look at the runtimes), though :-).

**Comment by chriseth** [comment]
The time for the build only (excluding setting up the docker image since the time for that can vary) goes from 8.5 minutes down to 3.5

**Pull request merged**